### ISSUE TITLE
Right-Clicking signs with dye requires trust

### ISSUE THREAD

**Issue by LiterallyRain** [GriefPrevention/GriefPrevention#316]
### What behaviour is observed:
Right clicking signs with dye in a claim requires trust (A minor issue with some shop plugins). Guessing it's identifying the sign as a sheep, item frame, or something of the like, considering they're all entities (Probably not sheeps, considering shears don't trigger a trust message).

### What behaviour is expected:
Expecting it to not trigger a trust restriction, as the right click holds no purpose on signs.

### Steps/models to reproduce:
Try to right click any sign with any dye (Bonemeal and Lapiz works) in a claim without trust and ignoreclaims. If you want to verify it's problematic nature, I have verified that it's a problem for "Shop - a simple, intuitive shop plugin" (Link to plugin: https://www.spigotmc.org/resources/shop-a-simple-intuitive-shop-plugin.9628/ )

### Spigot/Craftbukkit/Paper and GriefPrevention version:
Spigot: https://imgur.com/a/Sx7oPOb
Griefprevention: 16.7.1

### Stack trace/error
```
(I'm an administrator, but I don't have access to console for either of the requested files. However, as it's a simple non-server-specific issue it should not be necessary. (Unless we have a plugin that somehow adds an interaction between dye and signs, or rebrands the signs with an id similar to sheep/item frames, which I doubt)
```

### GriefPrevention config.yml (if applicable)
```
Not applicable
```

### Plugin list (if applicable):
(Probably) Not applicable

**Comment by Jikoo** [comment]
[This line](https://github.com/TechFortress/GriefPrevention/blob/master/src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java#L1771). Used to prevent bonemeal usage in claims.

**Comment by LiterallyRain** [comment]
Yes, but adding another "if sign" section or whatever to excempt signs from it shouldn't cause a problem, as it's got an actual effect on shop plugins. Shouldn't take long either. Not sure what effect it has with monster eggs, don't know if you can place mobs by right clicking on a sign, I have yet to test it.

**Comment by RoboMWM** [comment]
The interacteventhandler is the most organized place of code in this plugin. /s

Anyone can PR if they get to it before me, since I'm busy doing stupid life stuff this week. Probably something as stupid simple like yet another nested if `sign && dye` then return or whatever. Or just sign, if it is indeed not possible to spawn mobs via rightclick on sign with spawn egg.

**Comment by RoboMWM** [comment]
Should be fixed in 16.9 due to Minecraft's 1.13 "flattening" of block types.
